docs/create_webhook_fleetautoscaler.md

```diff
@@ -0,0 +1,231 @@
+# Quickstart Create a Fleet Autoscaler with Webhook Policy
+
+This guide covers how you can create webhook fleet autoscaler policy.
+The main difference from the Buffer policy is that the logic on how many target replicas you need is delegated to a separate pod.
+This type of Autoscaler would send an HTTP request to the webhook endpoint every sync period (which is currently 30s) with a JSON body, and scale the target fleet based on the data that is returned.
+
+## Prerequisites
+
+It is assumed that you have read the instructions to [Create a Game Server Fleet](./create_fleet.md)
+and you have a running fleet of game servers or you could run command from Step #1.
+
+## Objectives
+
+- Deploy the Webhook Pod and service for autoscaling
+- Create a Fleet Autoscaler with Webhook policy type in Kubernetes using Agones custom resource
+- Watch the Fleet scale up when allocating GameServers
+- Watch the Fleet scale down after GameServer shutdown
+
+### 1. Deploy the fleet
+
+Run a fleet in a cluster:
+```
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/master/examples/simple-udp/fleet.yaml
+```
+
+### 2. Deploy a Webhook service for autoscaling
+
+We need to create a pod which will handle HTTP requests with json payload `FleetAutoscaleReview` and return back it with `FleetAutoscaleResponse` populated.
```

Comment:
Should we add link those two payload type to the endpoint specifications ?

Reply:
I added them